packages-web/badge/src/Badge.xml

```diff
@@ -1,92 +1,66 @@
-            <description>Render it as either a badge or a color label</description>
-            <enumerationValues>
-                <enumerationValue key="badge">Badge</enumerationValue>
-                <enumerationValue key="label">Label</enumerationValue>
-                </enumerationValues>
-        </property>
-        <property key="onClickEvent" type="enumeration" defaultValue="doNothing">
-            <caption>On click</caption>
-            <category>Events</category>
-            <description/>
-            <enumerationValues>
-                <enumerationValue key="doNothing">Do nothing</enumerationValue>
-                <enumerationValue key="showPage">Show a page</enumerationValue>
-                <enumerationValue key="callMicroflow">Call a microflow</enumerationValue>
-                <enumerationValue key="callNanoflow">Call a nanoflow</enumerationValue>
-            </enumerationValues>
-        </property>
-        <property key="page" type="form" required="false">
-            <caption>Page</caption>
-            <category>Events</category>
-            <description>The page that will be opened on click</description>
-        </property>
-        <property key="openPageAs" type="enumeration" defaultValue="content">
-            <caption>Open page as</caption>
-            <category>Events</category>
-            <description/>
-            <enumerationValues>
-                <enumerationValue key="content">Full page</enumerationValue>
-                <enumerationValue key="popup">Popup</enumerationValue>
-                <enumerationValue key="modal">Blocking popup</enumerationValue>
-            </enumerationValues>
-        </property>
-        <property key="microflow" type="microflow" required="false">
-            <caption>Microflow</caption>
-            <category>Events</category>
-            <description>The microflow that will be executed on click</description>
-            <returnType type="Void" />
-        </property>
-        <property key="nanoflow" type="nanoflow" required="false">
-            <caption>Nanoflow</caption>
-            <category>Events</category>
-            <description>The nanoflow that will be executed on click</description>
-            <returnType type="Void" />
-        </property>
+        <propertyGroup caption="General">
+            <propertyGroup caption="General">
+                <property key="type" type="enumeration" required="true" defaultValue="badge">
+                    <caption>Type</caption>
+                    <description>Render it as either a badge or a color label</description>
+                    <enumerationValues>
+                        <enumerationValue key="badge">Badge</enumerationValue>
+                        <enumerationValue key="label">Label</enumerationValue>
+                    </enumerationValues>
+                </property>
+                <property key="defaultValue" type="textTemplate" required="false">
```

Comment:
`valueAttribute` property can be removed, as this can all be done with textTemplate value
`defaultValue` Change to `value` 
One case can not be implemented, with context, fallback to default value. The question is that used?

Reply:
Well, there is (in the near future) a fallback feature for text template. There are 3 options to go for:
1 ) Support v8.1.0 and keep value attribute in there.
2) Support v8.1.0 and remove value attribute -> no custom fallback value when context object isn't available. In version with the new fallback feature the custom fallback value is supported.
3) Support from the version that includes the fallback feature.

Reply:
Picked option 2. Feature of having a default value when the attribute is empty is gone.

Reply:
Done